cbindgen.toml

```diff
@@ -27,6 +27,10 @@ header = """
 # Warning to include saying the file is autogenerated.
 autogen_warning = "/* Warning, this file is autogenerated by cbindgen. Don't modify this manually. */"
 
+[defines]
+"target_family = unix" = "DEFINE_UNIX"
+"target_family = windows" = "DEFINE_WINDOWS"
```

Comment:
These are needed so that cbindgen wraps the conditional code in define guards in the generated header file.